pandas-stubs/core/arrays/categorical.pyi

```diff
@@ -64,22 +64,9 @@ class Categorical(ExtensionArray, PandasObject):
     ) -> Categorical: ...
     @property
     def codes(self) -> np_ndarray_int: ...
-    @overload
-    def set_ordered(self, value, inplace: Literal[True]) -> None: ...
-    @overload
-    def set_ordered(self, value, inplace: Literal[False] = ...) -> Categorical: ...
-    @overload
-    def set_ordered(self, value, inplace: bool = ...) -> Categorical | None: ...
-    @overload
-    def as_ordered(self, inplace: Literal[True]) -> None: ...
-    @overload
-    def as_ordered(self, inplace: Literal[False] = ...) -> Categorical: ...
-    @overload
-    def as_ordered(self, inplace: bool = ...) -> Categorical | None: ...
-    @overload
-    def as_unordered(self, inplace: Literal[True]) -> None: ...
-    @overload
-    def as_unordered(self, inplace: Literal[False] = ...) -> Categorical: ...
+    def set_ordered(self, value) -> Categorical: ...
+    def as_ordered(self) -> Categorical: ...
+    def as_unordered(self) -> Categorical: ...
```

Comment:
Could open a pandas PR - it still has those overloads.

Reply:
Opened one.